README.md

```diff
@@ -49,10 +50,9 @@ To update the table in the README:
 
 ## Recommendation
 
-If performance, correctness and interoperability are the most
-important factors, [gogoprotobuf](https://gogo.github.io/) is
-currently the best choice. It does require a pre-processing step (eg.
-via Go 1.4's "go generate" command).
+If correctness and interoperability are the most
+important factors, [gob](http://golang.org/pkg/encoding/gob/) and [json](http://golang.org/pkg/encoding/json/) do the job good.
```

Comment:
Gob isn't interoperable with anything except Go, so this doesn't make any sense. Protobuf is still the right option here, though not gogoprotobuf anymore because it's deprecated.

Reply:
I overlooked that, sorry. So I should replace protobuf with gob?

Reply:
Replace gob with protobuf please, yes.

Reply:
Okay, did that.